test/colony-network-auction.js

```diff
@@ -198,13 +189,21 @@ contract("ColonyNetworkAuction", accounts => {
         assert.isTrue(differenceQuantity.lte(errorMarginQuantity));
       });
     });
-  });
 
-  describe("when bidding", async () => {
-    beforeEach(async () => {
-      await tokenAuction.start();
+    it("should succeed if the last auction for the same token was started at least 30 days ago", async () => {
+      const previousAuctionStartTime = await tokenAuction.startTime.call();
+      // 30 days (in seconds)
+      await forwardTime(30 * 24 * 60 * 60, this);
+
+      await token.mint(quantity.toString());
+      await token.transfer(colonyNetwork.address, quantity.toString());
+      await colonyNetwork.startTokenAuction(token.address);
+      const newAuctionStartTime = await tokenAuction.startTime.call();
```

Comment:
Shouldn't be a need for the `.call()` here. I know we explicitly use `call` elsewhere in the file, but that's a relic from days gone by, and we've been trying to remove it where possible / not introduce more ones. Same on L194 as well.

Reply:
Oh I see. I changed the old `startTime.call()` usage on L104 too.